lib/server/services/qualityControlFlag/QcFlagService.js

```diff
@@ -650,6 +661,7 @@ class QcFlagService {
                 required: true,
             });
         }
+        // Else fething information for synchronous QC flags
```

Comment:
This comment is misleading, you should put it in the comment `MonALISA production, if not specified run will be fetched for synchronous QC flags`

Reply:
applied